lib/MusicBrainz/Server/Entity/URL/Muziekweb.pm

```diff
@@ -0,0 +1,22 @@
+package MusicBrainz::Server::Entity::URL::Muziekweb;
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name { 'Muziekweb' }
+
+__PACKAGE__->meta->make_immutable;
+no Moose;
+1;
+
+=head1 COPYRIGHT AND LICENSE
+
+Copyright (C) 2019 MetaBrainz Foundation
```

Comment:
You may want to put your own name or pseudo instead, that’s your code after all.
The project just require contributions to be under the same license terms, thanks.

Reply:
Nah, I waive all my personal rights for this code. I 'donate' it to the Metabrainz Foundation.

It's not very much my code imho, as I only changed the name, no creativity at all was needed.